plugins/postcss-cascade-layers/README.md

```diff
@@ -58,6 +58,62 @@ instructions for:
 | [Node](INSTALL.md#node) | [PostCSS CLI](INSTALL.md#postcss-cli) | [Webpack](INSTALL.md#webpack) | [Create React App](INSTALL.md#create-react-app) | [Gulp](INSTALL.md#gulp) | [Grunt](INSTALL.md#grunt) |
 | --- | --- | --- | --- | --- | --- |
 
+## Options
+
+### onRevertLayerKeyword
+
+The `onRevertLayerKeyword` option enables warnings if `revert-layer` is used.
+Transforming `revert-layer` for older browsers is not possible in this plugin.
+
+Defaults to `warn`
+
+```js
+postcssCascadeLayers({ onRevertLayerKeyword: 'warn' }) // 'warn' | false
+```
+
+```pcss
+/* [postcss-cascade-layers]: handling "revert-layer" is unsupported by this plugin and will cause style differences between browser versions. */
+@layer {
+	.foo {
+		color: revert-layer;
+	}
+}
+```
+
+### onMixedLayerOrder
```

Comment:
@romainmenke what do you think about a different name for this warning? At first glance, I wasn't quite sure what `mixed layer order` meant. Could something like `onMediaLayerOrder` work? Or maybe `onConditionalLayerUse`?

Reply:
> are we trying to capture that the media query could be changing the layer order?

yes :)

`onConditionalRulesChangingLayerOrder` this works for me!

Shall I update the code to match this?

Reply:
@romainmenke awesome and sure! I'm happy to make the change as well if you're busy, either way works!

Reply:
@romainmenke would you be able to give me permissions to push to the repo? I'm not able to push up my changes to your branch 😅

Reply:
I think we need to merge this branch first.
Then you will be able to apply the changes to the oddbird repo :)